docs/UserGuide.adoc

```diff
@@ -183,197 +183,282 @@ Example:
 // tag::addperson[]
 ==== Adding a person: `add`
 
-Adds a person to the address book. +
-Command format: `add n/NAME p/PHONE_NUMBER [p/ADDITIONAL_PHONE_NUMBERS]... e/EMAIL a/ADDRESS [f/] [dp/DISPLAY_PHOTO_FILE_PATH] [t/TAG]... [s/SOCIAL_PLATFORM USERNAME]...` +
-Alias: `a`
+To add a person to the address book, use the `add` command.
+
+====
+*Command format*: +
+`[red]#add# [green]#n/NAME p/PHONE_NUMBER [p/ADDITIONAL_PHONE_NUMBERS]... e/EMAIL a/ADDRESS [f/] [dp/DISPLAY_PHOTO_FILE_PATH] [t/TAG]... [s/SOCIAL_PLATFORM USERNAME]...#` +
+
+*Alias*: `a`
+====
 
 ****
```

Comment:
I think we should avoid using the **** format for more info about a command, since this is somewhat inconsistent with how we format descriptions for our other commands

Reply:
Updated ✅